_posts/2019-09-12-elixirconf-2019-review.md

```diff
@@ -0,0 +1,57 @@
+
+It was truly a pleasure to work with such a curious, dedicated and friendly group of students over the course of these two days. Thanks to everyone who signed up and participated! We had a great time teaching and we were challenged in turn by the interesting questions and approaches of everyone involved. A special shout out to our handful of participants who were brand new to Elixir and dove right into complex Phoenix concepts––successfully building out our application features by the end of the day and pushing and supporting the people around them with their open attitudes and fresh perspectives.
+
+If you'd like to learn more about these workshops, you can check out the code and slide deck for day one, as well as the code and slide deck for day two below.
+
+* Day 1: Harnessing the Real-Time Web with Phoenix, Channels, and Presence
+  * [Slide deck](https://speakerdeck.com/sophiedebenedetto/harnessing-the-real-time-web-with-phoenix-channels-plus-presence)
+  * Complete code
+* Day 2: Building Bulletproof Real-Time Applications with Phoenix, LiveView, and StreamData
+  * [Slide deck](https://speakerdeck.com/sophiedebenedetto/building-bulletproof-real-time-apps-with-phoenix-liveview-plus-stream-data)
+  * [Complete code for LiveView features](https://github.com/elixirschool/pointing-party/tree/live-view-js-hooks)
+  * [Complete code for Property-based testing features](https://github.com/elixirschool/pointing-party/tree/test-vote-calculation-with-stream-data)
+
+
+## Talks We Enjoyed
+
+There were so many exciting talks to check out, but with a multi-track schedule, we sadly couldn't catch them all. Here are a few that stood out to us in particular.
+
+### Jose Valim's Thursday Evening Keynote
+
+Just because Elixir is now feature-complete doesn't mean that the core team doesn't have some exciting and ambitious goals for the next year! Jose discussed the present and future of `Mix.Config`, including some thoughts on build time vs. run time configuration. He also made an exciting announcement--Phoenix 1.5 will ship with a native Telemetry module that makes it easy to configure your app's Telemetry metrics and report them to the third party service of your choosing (New Relic, Statsd, etc.). The Elixir community can expect a new version released every six months. One feature that we're excited for is ExUnit pattern diffing––making it easy to see exactly *what* doesn't match when ExUnit tells you that your match failed. Learn more about the future of Elixir by checking out the full talk [here](https://www.youtube.com/watch?v=oUZC1s1N42Q).
```

Comment:
It may be helpful to clarify that a new version of _Elixir_ will come out every six months. The way the sentence is now, it sounds like a new Phoenix version will come out every six months.

Reply:
good call. fixed!